docs/_includes/search/input.html

```diff
@@ -0,0 +1 @@
+<input type="text" id="docsearch-input" placeholder="Search the docs…">
```

Comment:
is there any usecase for including this somewhere else?

Reply:
I just wanted to isolate the component in its own file